pkg/assetfetcher/http_test.go

```diff
@@ -629,8 +628,56 @@ func TestMakeRequestSuccess(t *testing.T) {
 
 	wg.Add(1)
 	nexposeAssetFetcher.makeRequest(context.Background(), &wg, "siteID", 100, assetChan, errChan)
+	wg.Wait()
+	assert.NotNil(t, <-assetChan)
+}
+
+func TestMakeRequestSkipAndStatUnscannedAssets(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	mockRT := NewMockRoundTripper(ctrl)
+	mockStatFn := NewMockStat(ctrl)
```

Comment:
suggestion: This is not a function so the `Fn` suffix actually removes clarity. This should be `mockStat = NewMockStat(ctrl)`

Reply:
done!